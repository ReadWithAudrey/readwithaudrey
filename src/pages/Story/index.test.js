import React from 'react'
import renderer from 'react-test-renderer'
import Story from './index.js'

describe('Story page snapshot test', () =>
  it('renders correctly', () => {
    const tree = renderer.create(<Story />).toJSON()
    expect(tree).toMatchSnapshot()
  }))
