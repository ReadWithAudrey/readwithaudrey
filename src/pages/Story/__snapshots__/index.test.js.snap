// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Story page snapshot test renders correctly 1`] = `
Array [
  <header
    className="sc-bdVaJa flex items-center justify-between ph3 montserrat white bg-green w-100 f3 chiVga"
  >
    <div
      className="flex items-center"
      id="header-left"
    >
      <img
        alt="book icon"
        className="h2 ph2"
        src="test-file-stub"
      />
      <Link
        to="/"
      >
        Audrey
      </Link>
    </div>
    <Link
      className="w-third"
      to="/Form1/"
    >
      <button
        className="sc-bZQynM bg-green hover-bg-pink pa2 mv2 white br2 w-100 f5 montserrat krNSkU"
      >
        Join Us
      </button>
    </Link>
  </header>,
  <div
    className="flex flex-column items-center justify-center page-padding ph4"
  >
    <h1
      className="f2 pink tc montserrat mb3 mt4"
    >
      Our Story
    </h1>
    <p
      className="sc-jTzLTM montserrat f5 tc lh-copy dDfTnQ"
    >
      Audrey is based in London England, where I saw the benefits of peer-to-peer discussion when I used to be a high school teacher.
    </p>
    <p
      className="sc-jTzLTM montserrat f5 tc lh-copy dDfTnQ"
    >
      I've always been interested in literacy as a subject are and this, plus my interest in using technology for worthwhile causes, lead me to create Audrey. Audrey started in 2017 and has connected people from X amount of countries across the world.
    </p>
    <p
      className="sc-jTzLTM montserrat f5 tc lh-copy dDfTnQ"
    >
      We look forward to you becoming a part of our community and ultimately our story.
    </p>
  </div>,
  <footer
    className="sc-bwzfXH flex flex-column justify-center items-center montserrat bg-green w-100 bt bw2 b--green uTxCW"
  >
    <span
      className="sc-htpNat flex justify-around items-center bg-white black w-100  pa2 jbutSu"
    >
      Â© Audrey
      <Link
        className="no-underline black ph1"
        to="/terms/"
      >
        Terms
      </Link>
      <Link
        className="no-underline black ph1"
        to="/privacy/"
      >
        Privacy
      </Link>
      <Link
        className="no-underline black ph1"
        to="/contact/"
      >
        Contact
      </Link>
    </span>
  </footer>,
]
`;
